corpus/structure/root/METS_camelcase/README.md

```diff
@@ -0,0 +1,7 @@
+# IP for negative test-case
```

Comment:
I think we need something in the README metadata that indicates whether this is a pass or fail case. In time we'll want the expected error set I think, but for now: `**Valid**: true/false` will do. Naming convention for the test cases might be an alternative solution.